build.gradle.kts

```diff
@@ -77,6 +81,118 @@ java {
 }
 
 tasks {
+  val codeSearchCommit = "9d86a4f7d183e980acfe5d6b6468f06aaa0d8acf"
```

Comment:
what is code search? will we need to update this commit sometime?

Reply:
We have two products: Cody and Code Search. Code Search is implemented in the sourcegraph/sourcegraph repository, and this is a commit in that repo. This commit hash will need to be updated when we update code search features (alt-s).